interface/account.interface.ts

```diff
@@ -1,4 +1,4 @@
-export interface IUser {
+export interface IAccount {

```

Comment:
many fields are missing in this interface i.e companyId, tenantId, createdBy, updatedBy

Reply:
its not the valid interface... i will remove it